iso690-numeric-brackets-cs.csl

```diff
@@ -420,8 +420,15 @@
       <key variable="citation-number"/>
     </sort>
     <layout prefix="[" suffix="]" delimiter=", ">
-      <text variable="citation-number"/>
-      <text variable="locator" prefix=" s. "/>
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text variable="citation-number"/>
+        </group>
+        <group delimiter=" ">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </group>
     </layout>
```

Comment:
Yes, this is a better code as my. Thanks

2015-05-13 9:07 GMT+02:00 Philipp Zumstein <notifications@github.com>:

> In iso690-numeric-brackets-cs.csl
> <https://github.com/citation-style-language/styles/pull/1547#discussion_r30206638>
> :
>
> > @@ -420,8 +420,15 @@
> >        <key variable="citation-number"/>
> >      </sort>
> >      <layout prefix="[" suffix="]" delimiter=", ">
> > -      <text variable="citation-number"/>
> > -      <text variable="locator" prefix=" s. "/>
> > +      <group delimiter=", ">
> > +        <group delimiter=" ">
> > +          <text variable="citation-number"/>
> > +        </group>
> > +        <group delimiter=" ">
> > +          <label variable="locator" form="short"/>
> > +          <text variable="locator"/>
> > +        </group>
> > +      </group>
> >      </layout>
>
> You are right about the outer group, but the one-child-element-group seems
> unnecessary. If you want to have a space between citation-number and the
> other two variables, then how about:
>
> <layout prefix="[" suffix="]" delimiter=", ">
>   <group delimiter=" ">
>     <text variable="citation-number"/>
>     <group delimiter=" ">
>       <label variable="locator" form="short"/>
>       <text variable="locator"/>
>     </group>
>   </group>
> </layout>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1547/files#r30206638>
> .
>